src/core/formatCommaPositions.ts

```diff
@@ -1,7 +1,7 @@
 import type { CommaPosition } from 'src/types';
 import { maxLength } from 'src/utils';
 
-import { WHITESPACE_REGEX } from './Tokenizer';
+const WHITESPACE_REGEX = /^(\s+)/u;
```

Comment:
does this pattern also need the sticky flag ?

Reply:
Nope. This one I explicitly replaced to use a separate non-sticky regex, not import the sticky one from Tokenizer.